app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,8 @@ class RegistrationsController < Devise::RegistrationsController
   private
 
   def sign_up_params
-    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+    password = SecureRandom.hex(10)
```

Comment:
Creating a signup page just for admin is a) overkill, and b) a security risk. Create them in seeds or `rails c`.

You can keep an `edit` page so admins can update their password, but that should be it